bats/tests/registry/creds.bats

```diff
@@ -1,7 +1,12 @@
+setup_file() {
+    rm -f "$HOME/.docker/config.json"
```

Comment:
I wish there was a way to use the `CONFIG_FILE` setting globally, but it is probably not possible for the setup/teardown functions to share settings.